autogen/agentchat/conversable_agent.py

```diff
@@ -832,8 +832,10 @@ def _summarize_chat(
             if not isinstance(prompt, str):
                 raise ValueError("The summary_prompt must be a string.")
             msg_list = agent._groupchat.messages if hasattr(agent, "_groupchat") else agent.chat_messages[self]
-
-            summary = self._llm_response_preparer(prompt, msg_list, llm_agent=agent, cache=cache)
+            try:
+                summary = self._llm_response_preparer(prompt, msg_list, llm_agent=agent, cache=cache)
+            except Exception as e:
```

Comment:
What is the value of summary in the case of an error? Empty string? None? Will downstream agents handle that well?

Reply:
empty string (see line 823)